ignite/contrib/metrics/cohen_kappa.py

```diff
@@ -0,0 +1,131 @@
+from typing import Callable
+
+import torch
+
+from ignite.metrics import EpochMetric
+
+
+def non_weighted_cohen_kappa_compute_fn(y_preds: torch.Tensor, y_targets: torch.Tensor) -> float:
+    try:
+        from sklearn.metrics import cohen_kappa_score
+    except ImportError:
+        raise RuntimeError("This contrib module requires sklearn to be installed.")
+
+    y_true = y_targets.numpy()
+    y_pred = y_preds.numpy()
+    return cohen_kappa_score(y_true, y_pred, weights=None)
+
+
+class NonWeightedCohenKappa(EpochMetric):
```

Comment:
I think it would be more simple to implement a single class with an additional arg `weights`.
try/except check may be better to put into constructor (same should have been done for AveragePrecision, I think).

Reply:
But then the way the user uses this metrics it's going to be different from the other metrics

Reply:
You can implement it such that, weights is passed to `__init__` and before calling `super` with a function, you define this function according to `weights` value. Another way to do that is to wrap the function like
```python
def get_cohen_kappa_fn(weights):
    def wrapper(y_pred, y):
        y_true = y_targets.numpy()
        y_pred = y_preds.numpy()
        return cohen_kappa_score(y_true, y_pred, weights=weights)
    return wrapper
```